### PR TITLE
PTIR part1

### PR DESCRIPTION
**Pull request by qiuting1995**
I learned the part3.1 in the paper, which is about Image local Cost. It's a very important part for intelligent scissors because it decide how we define the edge of the image.
The next step for me is to learn part3.2, which is the algorithm 'Live-Wire 2-D DP graph search' and try to realize it in code.

### CODE REVIEW

**Comment by troussil** [comment]
Ok, but before merging, I want you to put your document into a specific directory (eg. scissors/doc)

**Comment by qiuting1995** [comment]
Morning,
Thank you for your letter. I have put them into a directory now:)


Yours,
QIU TING

2017-03-27 9:45 GMT+02:00 troussil <notifications@github.com>:

File: README.md [hunk 1]
```diff
@@ -1,2 +0,0 @@
-# segmentation
-Implémentation de plusieurs algorithmes de segmentation d'images
```
Comment (troussil) [inline README.md]: sorry, but you should not change this file. It describes the whole segmentation project (and you're not alone on this project). Please, revert your changes.

**Comment by qiuting1995** [comment]
I have changed the README back to the initial one.

**Pull request merged**